pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -105,6 +110,11 @@ type EbpfProgram struct {
 	// otherwise used to check if the program needs to be stopped and re-started
 	// when adding new libsets
 	isInitialized bool
+
+	// enabledProbes is a list of the probes that are enabled for the current system.
+	enabledProbes []manager.ProbeIdentificationPair
+	// disabledProbes is a list of the probes that are disabled for the current system.
+	disabledProbes []manager.ProbeIdentificationPair
```

Comment:
i saw a similar approach in other places in our code.
do you think it would make more sense to move that logic to the manager somehow.

it is a bit ridiculous for me to have so many ebpf manager wrappers and still duplicate code like this in the actual business logic of our ebpf products